safeeyes/config/lang/es.json

```diff
@@ -8,7 +8,7 @@
     },
     "exercises": {
         "short_break_close_eyes": "Cierra fuertemente tus ojos",
-        "short_break_roll_eyes": "Cierra tus ojos y mueve tus ojos en círculos con las manos",
+        "short_break_roll_eyes": "Cierra tus ojos y mueve tus ojos en círculos usando los dedos de las manos",
```

Comment:
Sorry, I misunderstood that exercise: I thought it consisted in moving the eyes in circles using the hand fingers, like I'd previously seen it in other "eye strain preventing" apps.
So, if it means looking at every side, at the maximum of it, it would be sth like: "Pon los ojos en blanco hacia cada lado"